### PR TITLE
Implemented end to end for applicant experience

### PR DESCRIPTION
**Pull request by sfhelmet**

### CODE REVIEW

File: Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx [hunk 1]
```diff
@@ -46,49 +52,81 @@ export const ProfilePageJob = () => {
...
-            setExperience(experience.slice(0, currentExperience).concat(experience.slice(currentExperience + 1, experience.length)));
-        else
+        if(isCandidate){
+            // setExperience(experience.slice(0, currentExperience).concat(experience.slice(currentExperience + 1, experience.length)));
+            delete_experience(bufferId);
+        }       
+        else{
             setJobPostings(jobPostings.slice(0, currentExperience).concat(jobPostings.slice(currentExperience + 1, jobPostings.length)));
+        }
+        window.location.reload();
```
Comment (DanielC2001) [inline Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx:62]: Why is this needed? Technically you're updating the state and you shouldn't need to manually reload the page

File: Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx [hunk 2]
```diff
@@ -46,49 +52,81 @@ export const ProfilePageJob = () => {
...
-                "description": bufferDescription
-            }]);
+                "description": bufferDescription,
+                //TODO: Add location and skills in the frontend
+                "location": "blank",
+                "skills": "blank",
+                "applicant": userId,
+            });
         }
         else{}
```
Comment (DanielC2001) [inline Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx:120]: This else condition encloses nothing! You sure you don't need to enclose the code after it?

File: Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx (hunk 1 above)
Comment (sfhelmet) [reply Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx:62]: I have decided to make the experience show in chronological order as it would make more sense. Im doing it in the backend. Out of laziness, I just refresh the page instead of relying on the frontend.

File: Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx (hunk 2 above)
Comment (sfhelmet) [reply Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx:120]: else{} is to prepare for next implementation for recruiters

**Review by DanielC2001 — APPROVED** [review]
Good to merge
